app/pages/modules/[suffix].tsx

```diff
@@ -318,6 +318,21 @@ const Module = ({ module, mainFile, supportingRaw }) => {
             ) : (
               ""
             )}
+            {/* Preview Office files */}
+            {mainFile.mimeType === "application/vnd.ms-excel" ||
+            mainFile.mimeType ===
+              "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" ||
+            mainFile.mimeType ===
+              "application/vnd.openxmlformats-officedocument.wordprocessingml.document" ? (
+              <iframe
```

Comment:
Assuming that we are supporting `.doc` file, do we need `application/msword`?

```suggestion
            mainFile.mimeType === "application/msword" || 
            mainFile.mimeType ===
              "application/vnd.openxmlformats-officedocument.wordprocessingml.document" ? (
              <iframe
```

Reply:
Thanks @nsunami! 

`.doc` and `.xls` are not open file formats, so we don't permit the uploading of those. 🦢 When we were setting up Hypergraph we decided to go the [open file format](https://en.wikipedia.org/wiki/List_of_open_file_formats) route, and it's been implemented in the same way here for longevity purposes.